spec/defines/vhost_spec.rb

```diff
@@ -57,4 +60,22 @@
       is_expected.to contain_file('www.example.com.crt').with(notify: 'Service[nginx]')
     }
   end
+
+  context 'with vault => true' do
+    let(:params) do
+      {
+        source_path: 'v1/api/kv/certs/puppet/',
+        vault: true
+      }
+    end
+
+    it {
```

Comment:
We should test some content as well, I’ll look up the exact matches names tomorrow but something like `.with_content(/TEST/)`, just to make sure we are creating the content as expected.